src/stan/services/variational/print_progress.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef STAN_SERVICES_VARIATIONAL_PRINT_PROGRESS_HPP
+#define STAN_SERVICES_VARIATIONAL_PRINT_PROGRESS_HPP
+
+#include <stan/services/io/do_print.hpp>
+#include <cmath>
+#include <iomanip>
+#include <iostream>
+#include <string>
+
+namespace stan {
+  namespace services {
+    namespace variational {
+
+      void print_progress(const int m,
```

Comment:
I'd suggest making changes to this one and opening up a new issue if you think it exists elsewhere. We've got to move our code to getting better, not just letting things slide because other parts are broken.